engines/sword25/package/packagemanager.cpp

```diff
@@ -198,6 +198,21 @@ byte *PackageManager::getFile(const Common::String &fileName, uint *fileSizePtr)
 	int bytesRead = in->read(buffer, in->size());
 	delete in;
 
+	// Modify the buffer to enable internal debugger if needed
+	if (debugChannelSet(-1, kDebugInternalDebugger) && fileName.matchString("/system/internal_config.lua")) {
```

Comment:
It  should be `fileName.equals()`.  `matchString` is for pattern, and you do not have pattern.